### PR TITLE
feat: [CGC-3] Refactoring ChartData file

### PR DESCRIPTION
**Pull request by henrikmv**
- Updated `GrowthChartBuilder.tsx`:
  - Modified the `scales` object to use translation for `timeUnit`.
- Deleted `ChartData.ts` file.
- Created `ChartDataZscores.ts`:
  - Defined datasets and metadata for various growth parameters for boys and girls in different age ranges.
- Renamed `wfh-girls/index.ts` to `wfhl-girls/index.ts` (simular changes for other files).
- Updated `chartDataTypes.ts`:
  - Added new interfaces and constants related to chart data and translations.

### CODE REVIEW

File: src/DataSets/newWhoStandardDataSets/ZScores/ChartDataZscores.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        datasetValues: wfl_g_0_2_y_z,
+        metadata: {
+          label: ChartCodes.wfl_g_0_2_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 45, end: 110 }
+        }
+      }
+    }
+  }
```
Comment (edvinstava) [inline src/DataSets/newWhoStandardDataSets/ZScores/ChartDataZscores.ts:207]: Can you take a look at the range. The range itself may be correct but the interval in the datasetValues are 0.5 and not 1, which leads to the graph only mapping through half the file

File: src/Plugin.types.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 export type EnrollmentOverviewProps = {
     programId: string;
-    orgUnitId: string;
+    orgtimeUnitId: string;
```
Comment (edvinstava) [inline src/Plugin.types.ts:4]: Revert back to orgUnitId
```suggestion
    orgUnitId: string;
```

**Review by edvinstava — COMMENTED** [review]
Well done Henrik, just a few comments that you can take a look at

File: src/types/chartDataTypes.ts [hunk 3]
```diff
@@ -1,11 +1,72 @@
...
+    lhfa_g_0_13_w_z: ChartLabelCodes.b_0_13_w,
+    lhfa_g_2_5_y_z: ChartLabelCodes.g_2_5_y,
+    wfa_b_0_5_y_z: ChartLabelCodes.b_0_5_y,
+    wfa_b_0_13_w_z: ChartLabelCodes.b_0_13_w,
+    wfa_g_0_5_y_z: ChartLabelCodes.g_0_5_y,
+    wfa_g_0_13_w_z: ChartLabelCodes.g_0_5_w,
+    wfh_g_2_5_y_z: ChartLabelCodes.g_2_5_y,
+    wfl_g_0_2_y_z: ChartLabelCodes.g_0_2_y,
+    wfh_b_2_5_y_z: ChartLabelCodes.b_2_5_y,
+    wfl_b_0_2_y_z: ChartLabelCodes.b_0_2_y
```
[the remark this answers is not in the document]
Comment (henrikmv) [reply src/types/chartDataTypes.ts:68]: There is actually a diferent between length and hight in this case, so i will leave it unchangesed for now.

[the remark this answers is not in the document]
Comment (henrikmv) [reply src/types/chartDataTypes.ts:68]: The indicator for the growth charts, is on the other hand named with lh

File: src/DataSets/newWhoStandardDataSets/ZScores/ChartDataZscores.ts (hunk 1 above)
Comment (henrikmv) [reply src/DataSets/newWhoStandardDataSets/ZScores/ChartDataZscores.ts:207]: Thank you!

**Review by edvinstava — COMMENTED** [review]
Well done Henrik, LGTM!

**Review by edvinstava — APPROVED** [review]

**Pull request merged**